### ISSUE TITLE
Error: Query Returned No Results (aws_ami)

### ISSUE THREAD

**Issue by BrewedCoffee** [hashicorp/learn-terraform-provision-eks-cluster#63]
Hello,

After performing `terraform plan`, I get the following error:

```
Error: Your query returned no results. Please change your search criteria and try again.
│ 
│   with module.eks.data.aws_ami.eks_worker[0],
│   on .terraform/modules/eks/data.tf line 20, in data "aws_ami" "eks_worker":
│   20: data "aws_ami" "eks_worker" {
│ 
```

Is the syntax outdated? I have installed all the prerequisites and updated modules/IAM credential access.

**Comment by judithpatudith** [comment]
Hi! Thanks for this report, we'll look into it!

**Comment by alanszlosek** [comment]
Hi @BrewedCoffee, can you provide some more information about your setup? I went through our EKS tutorial last week -- using the sample code from the tutorial repo as-is and Terraform v1.2.2 -- and was not able to reproduce the issue you had. Were you perhaps using a different region? Did you make any other changes to the Terraform configuration? Do you remember what version of Terraform you were using?

**Comment by alanszlosek** [comment]
This should now be fixed by #65. Please let us know if you are still having issues.
